i18n/en.json

```diff
@@ -2418,7 +2418,7 @@
   "announcement_bar.error.past_grace": "Enterprise license is expired and some features may be disabled. Please contact your System Administrator for details.",
   "announcement_bar.error.preview_mode": "Preview Mode: Email notifications have not been configured.",
   "announcement_bar.error.purchase_a_license_now": "Purchase a License Now",
-  "announcement_bar.error.site_url_gitlab.full": "Please configure your [site URL](https://docs.mattermost.com/administration/config-settings.html#site-url) either on the [System Console](/admin_console/environment/web_server) or, if you're using GitLab Mattermost, in gitlab.rb.",
+  "announcement_bar.error.site_url_gitlab.full": "Please configure your [site URL](https://docs.mattermost.com/administration/config-settings.html#site-url) either on the [System Console](/admin_console/environment/web_server) or, if you're using GitLab Mattermost, in Gitlab.rb.",
```

Comment:
Could you make sure that this is not an error? I don't believe the filename contains the capital `G`

Reply:
I fixed that!